### DISCUSSION TITLE
SSL Certificate verification failed

### DISCUSSION THREAD

**Discussion by Jss05** [sdwebui-w-horde/sd-webui-stable-horde-worker#98] [Q&A]
Hello everyone, I was trying to install the worker on my computer today, but after some other minor issues I finally got to turning it on. But when I did I immediately got this error message. After trying to figure out a fix myself and asking around the official stable horde discord server but we weren't able to figure it out.

![image](https://user-images.githubusercontent.com/52567460/236571930-1deec6f6-4b7e-4af7-b106-fff28df83250.png)

**Comment by MaikoTan** [comment]
This looks like issues on your network side or `stable horde`'s, although I could connect to `stablehorde.net` normally.
Are you using any VPN or something?

**Reply by Jss05** [reply]
No, I'm not using a VPN, and I'm also able to ping stablehorde.net completely fine, so it doesn't look to me to be a problem with my network. I've also been looking into the underlying plugin aiohttp, and it seems to be a reocurring problem for many users. I haven't been able to find a fix though.
